### PR TITLE
feat: [SRM-11022]: adding more Tests for templates

### PR DESCRIPTION
**Pull request by harshilbits**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by harshilbits** [comment]
retrigger titlecheck

**Comment by harshilbits** [comment]
retrigger jest

File: src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/__tests__/TemplatisedRunTimeMonitoredService.test.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                }
+              ]
+            }
+          } as VerifyStepMonitoredService['spec']['templateInputs']
+        }
+      },
+      expressions: ['org.identifier', 'org.name'],
+      allowableTypes: ['FIXED', 'RUNTIME', 'EXPRESSION'] as MultiTypeInputType[]
+    }
+    const { getByText } = render(<WrapperComponent {...props} />)
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/__tests__/TemplatisedRunTimeMonitoredService.test.tsx:76]: Component TemplatisedRunTimeMonitoredService has FormInput.MultiTypeInput
Should be wrap test component in formik and verify if values are populated as expected ?

Comment (harshilbits) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/__tests__/TemplatisedRunTimeMonitoredService.test.tsx:76]: Yeah , that can be an additional test. Main idea was depending on the values , correct fields should get populated.

Comment (deepesh-ui) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/__tests__/TemplatisedRunTimeMonitoredService.test.tsx:76]: lets plan for this for upcoming PR's

**Review by deepesh-ui — APPROVED** [review]

**Review by reetika04 — APPROVED** [review]

**Pull request merged**